core-bundle/src/Search/Backend/Provider/TagProvidingProviderInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Search\Backend\Provider;
+
+/**
+ * @experimental
+ */
+interface TagProvidingProviderInterface
```

Comment:
```suggestion
interface TagProviderInterface
```

Maybe shorter? Feel free to ignore 😊

Reply:
It's not a "tag provider". It's a "search provider that also provides tags" :) So I think the current naming is longer but also more "correct".

Reply:
Just a thought: Is the provider really providing tags? Or can it rather generate facet labels for existing tags? I'm thinking about something like `TagResolverInterface` or `TagSupportingInterface`, maybe even without the word "provider"?

So that a provider class would effectively `implement ProviderInterface, FooCapableInterface, BarSupportingInterface`. But I'm also fine with the current solution. :relaxed: